azure/create-cluster.sh

```diff
@@ -62,5 +62,4 @@ connection_string=$(echo ${connection_string} | cut -d "\"" -f 2)
 
 echo "run './connect.sh' to connect to the coordinator, or ALTERNATIVELY:"
 
-echo "run './delete-security-rule.sh' to temporarily disable security rule, and connect with:"
 echo ${connection_string}
```

Comment:
```suggestion
echo "and connect with: ${connection_string}"
```

Reply:
I prefer `echo ${connection_string}` as it makes it easy to copy & paste.

> ALTERNATIVELY:

Maybe `ALTERNATIVELY RUN THE FOLLOWING:`

Reply:
@onurctirtir FYI I changed this as `ALTERNATIVELY RUN THE FOLLOWING:` as it can indeed easier for copy & paste, I thought it was okay for you since you approved, let me know if it is not